### ISSUE TITLE
Segfault upon read

### ISSUE THREAD

**Issue by baasman** [IntelLabs/vdms#38]
This morning I kept running into the following error when using the following query: 
```
    queries = [
        {
            'FindEntity': {
                '_ref': 1,
                'class': 'Study',
                'constraints': {
                    'AccessionNumber': ['==', accession_number],
                },
            },
        },
        {
            'FindEntity': {
                '_ref': 2,
                'class': 'Series',
                'link': {
                    'ref': 1
                }
            },
        },
        {
            'FindImage': {
                'link': {
                    'ref': 2
                },
            }
        }
    ]
```

```
Oct 10 10:15:38 ykmits1 chdi_run_server.sh: Connection received...
Oct 10 10:15:38 ykmits1 chdi_run_server.sh: ch_tx_total: 32 [ms]
Oct 10 10:15:38 ykmits1 chdi_run_server.sh: ch_tx_query: 32 [ms]
Oct 10 10:15:38 ykmits1 chdi_run_server.sh: ch_tx_send: 0 [ms]
**Oct 10 10:15:38 ykmits1 kernel: vdms[23044]: segfault at 1 ip 00007f3a84d60bfa sp 00007f3a77c9ce38 error 4 in libpmgd.so[7f3a84d25000+60000]**
Oct 10 10:15:38 ykmits1 chdi_run_server.sh: /data/vdms/vdms/chdi_run_server.sh: line 32: 23043 Segmentation fault      ./vdms 2>> $VDM_LOG_ERR
Oct 10 10:15:38 ykmits1 systemd: vdms.service: main process exited, code=exited, status=139/n/a
Oct 10 10:15:38 ykmits1 systemd: Unit vdms.service entered failed state.
Oct 10 10:15:38 ykmits1 systemd: vdms.service failed.
```
However, after rebuilding the `db` directory the problem went away.

**Comment by luisremis** [comment]
We will include this fix soon, it is related to #12 
Thanks!

**Comment by luisremis** [comment]
Fix in #71
